R/pkg/R/stats.R

```diff
@@ -134,9 +129,7 @@ setMethod("freqItems", signature(x = "SparkDataFrame", cols = "character"),
             collect(dataFrame(sct))
           })
 
-#' approxQuantile
-#'
-#' Calculates the approximate quantiles of a numerical column of a SparkDataFrame.
+#' approxQuantile - Calculates the approximate quantiles of a numerical column of a SparkDataFrame.
```

Comment:
Oh, @shivaram already mentioned this. +1 for @shivaram 's opinion.
If we delete the details, `@description` is not needed.